lib/std/fs.zig

```diff
@@ -300,6 +300,7 @@ pub const Dir = struct {
             buf: [8192]u8, // TODO align(@alignOf(os.system.dirent)),
             index: usize,
             end_index: usize,
+            first: bool,
```

Comment:
Would you mind renaming the field to something less ambiguous like for example `first_scan` or `first_iteration`? IMHO this will greatly improve readability of this code.

Reply:
Was doing this to match the existing Windows implementation, so should that be changed also?

Reply:
Yeah, change the field name in all children. I think in this case being verbose is a good thing.